bloomrun.js

```diff
@@ -152,4 +152,14 @@ Bloomrun.prototype.iterator = function (obj, opts) {
   return new Iterator(this, obj, asMatch)
 }
 
+Bloomrun.prototype[Symbol.iterator] = function () {
+  var iterator = this.iterator()
+  return {
+    next: function () {
+      var result = iterator.next()
+      return (result !== null) ? { value: result, done: false } : { done: true }
+    }
+  }
```

Comment:
you can just call iterator[Symbol.iterator]() here and reuse the code in the Iterator itself.

Reply:
Oops, didn't notice that.